modelkit/assets/manager.py

```diff
@@ -44,23 +44,11 @@ def __init__(self, **settings):
         self.timeout = settings.timeout
 
         self.remote_assets_store = None
-        if settings.remote_store:
-            try:
-                self.remote_assets_store = RemoteAssetsStore(
-                    **settings.remote_store.dict()
-                )
-                logger.debug(
-                    "AssetsManager created with remote storage provider",
-                    driver=self.remote_assets_store.driver,
-                )
-            except BaseException:
-                # A remote store was parametrized, but it could not be instantiated
-                logger.error(
-                    "Failed to instantiate the requested remote storage provider"
-                )
-                raise
-        else:
-            logger.debug("AssetsManager created without a remote storage provider")
+        self.remote_assets_store = RemoteAssetsStore(**settings.remote_store.dict())
+        logger.debug(
+            "AssetsManager created with remote storage provider",
+            driver=self.remote_assets_store.driver,
+        )
```

Comment:
These lines do more than only improving assetsManager error handling, these change its behavior since we can have `settings.remote_store` set to False or None, and only use a local storage.

In the main version, we log + re-raise the exception, which seems more explicit than the proposed enhancement (?)

I am not sure on that one :)

Reply:
This is probably what breaks the tests. We rely on whether `self.remote_assets_store` is None to decide where to look for assets